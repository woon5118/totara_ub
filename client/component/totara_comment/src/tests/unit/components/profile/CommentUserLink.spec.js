/**
 * This file is part of Totara Enterprise Extensions.
 *
 * Copyright (C) 2020 onwards Totara Learning Solutions LTD
 *
 * Totara Enterprise Extensions is provided only to Totara
 * Learning Solutions LTD's customers and partners, pursuant to
 * the terms and conditions of a separate agreement with Totara
 * Learning Solutions LTD or its affiliate.
 *
 * If you do not have an agreement with Totara Learning Solutions
 * LTD, you may not access, use, modify, or distribute this software.
 * Please contact [licensing@totaralearning.com] for more information.
 *
 * @author Kian Nguyen <kian.nguyen@totaralearning.com>
 * @module totara_comment
 */

import UserLink from 'totara_comment/components/profile/CommentUserLink';
import { shallowMount } from '@vue/test-utils';

describe('totara_comment/components/profile/CommentUserLink.vue', () => {
  let wrapper = null;

  beforeAll(() => {
    wrapper = shallowMount(UserLink, {
      propsData: {
        userFullName: 'Admin user',
        profileUrl: 'http://example.com',
      },
    });
  });

  it('Checks snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
