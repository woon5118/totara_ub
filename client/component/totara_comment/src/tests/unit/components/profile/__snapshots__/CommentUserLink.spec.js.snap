// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`totara_comment/components/profile/CommentUserLink.vue Checks snapshot 1`] = `
<a
  class="tui-commentUserLink tui-commentUserLink--smallLink"
  href="http://example.com"
>
  
  Admin user

</a>
`;
